sys/posix/include/semaphore.h

```diff
@@ -87,7 +64,7 @@ int sem_wait(sem_t *sem);
  * @brief WARNING: currently not supported
  *
  * @param sem Semaphore to wait on
- * @param abstime Max time to wait for a post
+ * @param abstime Absolute time to wait for a post
```

Comment:
This is still confusing me, is this supposed to be a point in time in the future or is it supposed to be the length of the interval from now until when we give up waiting?